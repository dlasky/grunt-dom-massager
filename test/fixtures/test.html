<!DOCTYPE html>
<html>
	<head>
		<title>DOM TEST</title>
	</head>
	<body>

		<div class="read">
			<div class="classy"><span>i'm classy</span></div>
			<div>i'm not</div>
			<div id="idface">i have an id</div>
			<div>i don't</div>
			<div id="attrtest" monkeys="true" victory="atSea">
			</div>
		</div>

		<div class="write">
			<div id="deleteme">i'd like to be deleted</div>
			<div id="after">append something after me</div>
			<div id="before">prepend something before me</div>
		</div>

	</body>
</html>