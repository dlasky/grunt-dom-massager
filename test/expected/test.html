<!DOCTYPE html>
<html>
	<head>
		<title>DOM TEST</title>
	</head>
	<body>

		<div class="read">
			<div class="classy"><span>i'm classy</span></div>
			<div>i'm not</div>
			<div id="idface">i have an id</div>
			<div>i don't</div>
			<div id="attrtest" monkeys="true" victory="atSea">
			</div>
		</div>

		<div class="write">
			
			<div id="after">append something after me<a href="blah"></a></div>
			<div id="before"><a href="zappo"></a>prepend something before me</div>
		</div>

		<div class="attr">
			<div id="1" class="test test2"></div>
			<div id="2" class="toast"></div>
			<div id="3" class=""></div>
			<div id="4" data-test="testFace"></div>
		</div>

	</body>
</html>
